src/types/index.ts

```diff
@@ -224,7 +224,47 @@ type ComparatorInput = {
   requestMetadata: object;
   feature: string;
 };
+type SourceDefinition = {
+  ID: string
+  Name: string
+  Category: string
+  Type: string
+}
+type TrackingPlanT = {
```

Comment:
Do you need these structures?

Reply:
Currently they are not used but since we are importing them in source object I have maintained their structure

Reply:
If they are not used, can we remove them?

Reply:
Removed